NuGet.Services.Dashboard.Operations/Infrastructure/SecretConfigurationManager.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Configuration;
+using NuGet.Services.KeyVault;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace NuGet
+{
+    public static class SecretConfigurationManager
+    {
+        private static ISecretInjector _secretInjector;
+        private static ISecretReader _keyVaultReader;
+
+        public static void Init()
+        {
+            var vaultName = ConfigurationManager.AppSettings["VaultName"];
+            var clientId = ConfigurationManager.AppSettings["ClientId"];
+            var certificateThumbprint = ConfigurationManager.AppSettings["CertificateThumbprint"];
+            var validateCertificate = ConfigurationManager.AppSettings["ValidateCertificate"].Equals("true", StringComparison.InvariantCultureIgnoreCase) ? true : false;
```

Comment:
Good point.

---
In reply to: [71935434](https://github.com/NuGet/NuGet.Services.Dashboard/pull/187#discussion_r71935434) [](ancestors = 71935434)